firebase-ai/src/main/kotlin/com/google/firebase/ai/common/APIController.kt

```diff
@@ -191,12 +191,7 @@ internal constructor(
       throw FirebaseCommonAIException.from(e)
     }
 
-  private fun HttpRequestBuilder.applyCommonConfiguration(request: Request) {
```

Comment:
Since the request `when` does not include any provision for bidi, do we really need to split them? would it be enough to apply the common configuration to bidi?

Reply:
scrap that, it doesn't have a `request` object to send. LGTm